ibi/coreos-installer/scripts/install-rhcos-and-restore-seed.sh

```diff
@@ -33,4 +35,4 @@ EOF
 systemctl start crio
 
 # To be changed when ibi-seed-install binary is created
-/usr/local/bin/ostree-restore.sh $seed_image
+/usr/local/bin/ostree-restore.sh $seed_image $seed_version $lca_image
```

Comment:
Lets move the logics that remain in the ostree-restore.sh script to this one and get rid of ostree-restore.sh once and for all :)